scripts/main/album.js

```diff
@@ -298,6 +302,100 @@ album.add = function (IDs = null, callback = null) {
 };
 
 
+album.addByTags = function (IDs = null, callback = null) {
+
+	const action = function (data) {
+
+		basicModal.close();
+
+		let params = {
+			title: data.title,
+			tags: data.tags,
+			parent_id: 0
+		};
+
+		api.post('Album::addByTags', params, function (_data) {
+			const isNumber = (n) => (!isNaN(parseInt(n, 10)) && isFinite(n));
```

Comment:
I'm sure @ildyria loves this syntax. He's been itching to use something like this on the server side :wink:.